src/System.ServiceModel.Primitives/tests/ServiceModel/DuplexClientBaseTest.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Threading.Tasks;
+using Xunit;
+
+public class DuplexClientBaseTest
+{
+    [Fact]
+    public static void CreateDuplexClientBase()
+    {
```

Comment:
Nit: Close is not in contract, but oddly Abort is.  Recommend rename DuplexClientBase_Ctor_Initializes_State and just validate the properties and any other reachable state. Then make a separate test DuplexClientBase_Aborts.